server/index.js

```diff
@@ -6,6 +6,29 @@ const port = 8000;
 const mysql = require("mysql2");
 const connection = mysql.createConnection(process.env.DATABASE_URL);
 
+// Session storage imports
+const passport = require('passport');
+var session = require('express-session');
+var SQLiteStore = require('connect-sqlite3')(session);
+
+// Import passport configuration
+require("./src/configs/passport");
+
+// So that we can send and receive JSON through express
+app.use(express.json());
+
+// Import router for all authentication API endpoints
+const authRouter = require('./src/routes/auth');
+
+// Set up session for authorization
+app.use(session({
+  secret: 'blueprint-tces',
```

Comment:
This should be moved to the .env file (and changed) -- the secret should not be available publicly. For example you can generate it using the crypto module you used earlier. Running something like this crypto.randomBytes(32).toString('hex') should do.

Reply:
Done!